mail_template_multi_report/tests/test_email_template.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests import common
+
+
+class TestEmailTemplate(common.TransactionCase):
+
+    def setUp(self):
+        super(TestEmailTemplate, self).setUp()
+
+        self.report_view = self.env['ir.ui.view'].create({
+            'name': 'test_report_template',
+            'mode': 'primary',
+            'type': 'qweb',
+            'arch': """\
+<?xml version="1.0"?>
+<t t-name="mail_template_multi_report.test_report_template">
+    <t t-call="report.html_container">
+        <t t-call="report.internal_layout">
+            <div class="page">
+            </div>
+        </t>
+    </t>
+</t>
+            """
```

Comment:
:grin: true, but demo data also allows for easier and faster runbot functional testing.